src/test/java/xyz/vegaone/easytracking/project/ProjectServiceTest.java

```diff
@@ -28,12 +28,16 @@
     public static final String PROJECT_NAME="Project name";
     public static final String PROJECT_NEW_NAME="Project new name";
 
-    @Autowired
     private ProjectService projectService;
 
-    @Autowired
     private UserStoryService userStoryService;
 
+    @Autowired
+    public ProjectServiceTest(ProjectService projectService, UserStoryService userStoryService) {
+        this.projectService = projectService;
+        this.userStoryService = userStoryService;
+    }
+
```

Comment:
This class has some unused imports and commented code as well. Could you please remove them?